src/topology/homeomorph.lean

```diff
@@ -504,7 +504,6 @@ variables [decidable_eq ι] (i : ι)
   continuous_inv_fun := continuous_pi $ λ j, by { dsimp only [equiv.pi_split_at],
     split_ifs, subst h, exacts [continuous_fst, (continuous_apply _).comp continuous_snd] } }
 
-variable (β)
```

Comment:
I did it to remove the tag in case that made it easier to close this PR, I'll just revert it since it was already easy to port.
Should the abbreviations stay? I think the abbreviation for the inverse makes some lemmas easier to parse for a human (eg. `symm_spec`)

Reply:
Yes, I am fine with keeping the `split_at`, `insert_at` abbreviations.